standard/expressions.md

```diff
@@ -2254,9 +2316,17 @@ The binding-time processing of an indexer access of the form `P[A]`, where `P` i
   - If `I` is applicable with respect to `A` ([§12.6.4.2](expressions.md#12642-applicable-function-member)) and `S` is a class type other than `object`, all indexers declared in an interface are removed from the set.
 - If the resulting set of candidate indexers is empty, then no applicable indexers exist, and a binding-time error occurs.
 - The best indexer of the set of candidate indexers is identified using the overload resolution rules of [§12.6.4](expressions.md#1264-overload-resolution). If a single best indexer cannot be identified, the indexer access is ambiguous, and a binding-time error occurs.
-- The index expressions of the *argument_list* are evaluated in order, from left to right. The result of processing the indexer access is an expression classified as an indexer access. The indexer access expression references the indexer determined in the step above, and has an associated instance expression of `P` and an associated argument list of `A`, and an associated type that is the type of the indexer. If `T` is a class type, the associated type is picked from the first declaration or override of the indexer found when starting with `T` and searching through its base classes.
+- The accessors of the best indexer are checked:
+  - If the indexer access is the target of an assignment then the indexer shall have a set or ref get accessor, if not a binding-time error occurs;
```

Comment:
Not thrilled about the comma here either, but within the context it's harder to fix.

Reply:
How about this?
```suggestion
  - If the indexer access is the target of an assignment then the indexer shall have a set or ref get accessor. if not, a binding-time error occurs.
```

Reply:
Changed to “, otherwise,” in next commit